README.md

```diff
@@ -0,0 +1,5 @@
+# data-driven-control
+[`data-driven-control`](https://github.com/fdcl-data-driven-control/data-driven-control) is a repository for studying and doing research about data-driven control algorithms for theoretical developments and practical applications to aerospace engineering.
+
```

Comment:
## Submodules 
- [DataDrivenControl.jl](https://github.com/fdcl-data-driven-control/DataDrivenControl.jl) : A julia package for data-driven-control.